lib/SearchAndSort/SearchAndSort.js

```diff
@@ -16,6 +17,7 @@ import {
   noop,
   isEmpty,
   defer,
+  omit,
```

Comment:
Note that `omit` is, idiotically, slated for removal from Lodash. I suppose we'll just wait till that happens, then bulk-port all our uses of it to whatever `lodash-omit-replacement` library emerges at that point.

(To be clear, this is not really a comment on your perfectly cromulent code, just a whinge about the state of the world.)

Reply:
Sorry it took so long to get to this, by the way!